python/erdos/context.py

```diff
@@ -44,19 +50,19 @@ def __init__(
         self,
         timestamp: PyTimestamp,
         config: "erdos.OperatorConfig",
-        write_stream: WriteStream,
+        write_stream: WriteStream[U],
```

Comment:
Can you also change the documentation above so that Sphinx can show the types? I believe they get the types automatically from the function signatures now so it might be time to remove `:py:class` attributes from documentation. I'm not sure though, but worth checking out.